wallet/how-to/connect/detect-wallet/multiple-wallets.md

```diff
@@ -0,0 +1,248 @@
+
+  :::note
+  MetaMask SDK does not support connecting to non-MetaMask wallets via EIP-6963.
+  If you intend to support discovery of other wallets, we recommend using other methods of adding
+  EIP-6963 support such as Wagmi 2+.
+  :::
+
+- [Wagmi 2+](https://wagmi.sh)
+
+- [Web3Modal 3+](https://docs.walletconnect.com/web3modal/about)
+
+- [MIPD Store](https://github.com/wevm/mipd)
+
+## Implement EIP-6963
+
+You can directly implement support for EIP-6963 in your React dapp.
+The steps are as follows:
+
+### 1. Review the EIP-6963 interfaces
+
+Review and understand the interfaces implemented and exposed by wallets that support EIP-6963:
+
+#### Provider info
+
+The [`EIP6963ProviderInfo`](https://eips.ethereum.org/EIPS/eip-6963#provider-info) interface
+represents the assets needed to display a wallet:
+
+```typescript
+interface EIP6963ProviderInfo {
+  uuid: string;
+  name: string;
+  icon: string;
+  rdns: string;
+}
+```
+
+#### Provider detail
+
+The [`EIP6963ProviderDetail`](https://eips.ethereum.org/EIPS/eip-6963#provider-detail) interface
+represents additional metadata about the wallet:
+
+```typescript
+interface EIP6963ProviderDetail {
+  info: EIP6963ProviderInfo;
+  provider: EIP1193Provider;
+}
+```
+
+#### Announce and request events
+
+The [`EIP6963AnnounceProviderEvent`](https://eips.ethereum.org/EIPS/eip-6963#announce-and-request-events)
+interface announces an event dispatched by the wallet:
+
+```typescript
+interface EIP6963AnnounceProviderEvent extends CustomEvent {
+  type: "eip6963:announceProvider";
+  detail: EIP6963ProviderDetail;
+}
+```
+
+The [`EIP6963RequestProviderEvent`](https://eips.ethereum.org/EIPS/eip-6963#announce-and-request-events)
+interface requests an event dispatched by a dapp:
+
+```typescript
+interface EIP6963RequestProviderEvent extends Event {
+  type: "eip6963:requestProvider";
+}
+```
+
+### 2. Set up a Vite project
+
+Set up a Vite project and update `src/vite-env.d.ts` with the EIP-6963 interfaces:
+
+```typescript title="vite-env.d.ts"
+/// <reference types="vite/client" />
+
+interface EIP6963ProviderDetail {
+  info: EIP6963ProviderInfo;
+  provider: EIP1193Provider;
+}
+
+interface EIP6963ProviderInfo {
+  walletId: string;
+  uuid: string;
+  name: string;
+  icon: string;
+}
+
+type EIP6963AnnounceProviderEvent = {
+  detail:{
+    info: EIP6963ProviderInfo,
+    provider: EIP1193Provider
+  }
+}
+
+interface EIP1193Provider {
+  isStatus?: boolean;
+  host?: string;
+  path?: string;
+  sendAsync?: (request: { method: string, params?: Array<unknown> }, callback: (error: Error | null, response: unknown) => void) => void
+  send?: (request: { method: string, params?: Array<unknown> }, callback: (error: Error | null, response: unknown) => void) => void
+  request: (request: { method: string, params?: Array<unknown> }) => Promise<unknown>
+}
+```
+
+:::note
+In addition to the EIP-6963 interfaces, you need the `EIP1193Provider` interface, defined by
```

Comment:
Maybe this instead...

In addition to the EIP-6963 interfaces, you need the `EIP1193Provider` interface (defined by
[EIP-1193](https://eips.ethereum.org/EIPS/eip-1193)), which is the foundational structure for
Ethereum wallet providers. This structure represents the essential properties and methods for interacting with dapps.

Not sure if I got this right. Was trying to make all this info a little more digestible.